activemodel/lib/active_model/type/value.rb

```diff
@@ -2,9 +2,15 @@
 
 module ActiveModel
   module Type
+    # The base class for all attribute types, as well as the default attribute
+    # type when a cast type is not specified when defining an attribute.
     class Value
       attr_reader :precision, :scale, :limit
 
+      # Define the type instance's state with three basic configuration
+      # settings: precision, limit, and scale. These values are dependent on the
+      # type subclass specifics. In the Value base class they are used for
+      # instance comparison and hash representation only.
```

Comment:
Hmm, what do you think about...?

```suggestion
      # Initializes a type with three basic configuration settings: precision, 
      # limit, and scale. The Value base class does not define behavior for 
      # these settings. It uses them for equality comparison and hash key 
      # generation only.
```